pkg/operator/ceph/csi/csi.go

```diff
@@ -124,6 +125,8 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 		return errors.Wrap(err, "failed to parse value for 'CSI_ENABLE_LIVENESS'")
 	}
 
+	CSIParam.HostPathRequiresPrivileged = controller.HostPathRequiresPrivileged()
```

Comment:
update the comment on ROOK_HOSTPATH_REQUIRES_PRIVILEGED in yamls/doc to convey that its now used for csi as well.